tensorflow/core/ops/math_ops.cc

```diff
@@ -478,6 +478,14 @@ REGISTER_OP("Ceil")
 Returns element-wise smallest integer in not less than x.
 )doc");
 
+REGISTER_OP("Rint")
+    .Input("x: T")
+    .Output("y: T")
+    .Attr("T: {float, double}")
+    .Doc(R"doc(
+Returns element-wise integer closest to x.
```

Comment:
I'm on the fence about whether setting the flags is worth the trouble.  If someone sets the processor flags I'm okay making them suffer the consequences.